Flake unit test

```
  	github.com/openshift/odo/pkg/url/labels	1.011s	coverage: 100.0% of statements
--- FAIL: TestRemoveDuplicate (0.00s)
    --- FAIL: TestRemoveDuplicate/Case_2_-_Remove_duplicates,_none_in_array (0.00s)
        util_test.go:1069: expected [bar foo], got [foo bar]
```
Seems like the output list changes order and thus the error 
https://github.com/openshift/odo/pull/1779#issuecomment-498612277

--- a/pkg/util/util_test.go
+++ b/pkg/util/util_test.go
@@ -9,6 +9,7 @@ import (
 	"path/filepath"
 	"reflect"
 	"regexp"
+	"sort"
 	"testing"
 
 	"github.com/pkg/errors"
@@ -1055,7 +1056,7 @@ func TestRemoveDuplicate(t *testing.T) {
 			name: "Case 2 - Remove duplicates, none in array",
 			args: args{
 				input:  []string{"bar", "foo"},
-				output: []string{"bar", "foo"},
+				output: []string{"foo", "bar"},
 			},
 		},
 	}
@@ -1064,6 +1065,10 @@ func TestRemoveDuplicate(t *testing.T) {
 
 			// Run function RemoveDuplicate
 			output := RemoveDuplicates(tt.args.input)
+
+			// sort the strings
+			sort.Strings(output)
+			sort.Strings(tt.args.output)
 
 			if !(reflect.DeepEqual(output, tt.args.output)) {
 				t.Errorf("expected %v, got %v", tt.args.output, output)

